Dockerfile.v2

```diff
@@ -0,0 +1,50 @@
+FROM radixdlt/babylon-node-build-layers:v1.3.0-java AS java-container
+
+WORKDIR /radixdlt
+
+# Copy the relevant files at the repo root
+COPY \
+  build.gradle \
+  gradlew \
+  gradlew.bat \
+  settings.gradle \
+  sonar-project.properties \
+  gradle.properties \
+  licence-header.txt \
+  /radixdlt/
+COPY ./gradle /radixdlt/gradle
+COPY ./common /radixdlt/common
+COPY ./core /radixdlt/core
+COPY ./core-rust-bridge /radixdlt/core-rust-bridge
+COPY ./cli-tools /radixdlt/cli-tools
+COPY ./shell /radixdlt/shell
+COPY ./keygen /radixdlt/keygen
+# Need .git for tag versions - but this can probably be removed soon
+COPY ./.git/* /radixdlt/.git/
```

Comment:
I'm fairly certain this can now be removed after Les's changes ~8 months ago :+1: